e2e/cypress/integration/ui/settings/project_mgmt.spec.ts

```diff
@@ -75,171 +77,190 @@ describe('project management', () => {
 
     cy.get('[data-cy=save-button]').click();
     cy.get('app-project-list chef-modal').should('not.be.visible');
-    cy.get('#main-content-wrapper').scrollTo('top');
-    cy.get('app-notification.info').should('be.visible');
+
+    // verify success notification and then dismiss it
+    cy.get('app-notification.info').contains(`Created project ${projectID}`);
+    cy.get('app-notification.info chef-icon').click();
+
     cy.contains(projectName).should('exist');
     cy.contains(projectID).should('exist');
 
     cy.url().should('include', '/settings/projects');
   });
 
-  it('addPolicies checkbox is always checked on modal open', () => {
```

Comment:
this is now just an assertion in `can create a project with a custom ID and generate associated policies`